exercises/high-scores/runtests.jl

```diff
@@ -0,0 +1,38 @@
+# canonical data version 4.0.0
+
+using Test
+
+include("high-scores.jl")
+
+
+@testset "List of scores" begin
+    @test [30, 50, 20, 70] == [30, 50, 20, 70]

```

Comment:
The canonical data suggests it should test the `scores` property. In its current form it doesn't test anything useful.